gateway-ha/pom.xml

```diff
@@ -187,6 +187,12 @@
             <version>1.70</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.codehaus.janino</groupId>
+            <artifactId>janino</artifactId>
+            <version>${dep.janino.version}</version>
+        </dependency>
```

Comment:
I don't feel much benefit to replace the existing dependency with new one. We're going to use airlift as the base plugin that uses JUL for logger.

Reply:
Just wondering why is trino using JUL for logger? Looks like slf4j is more powerful than JUL